content/rc/administration/setup/create-database.md

```diff
@@ -56,16 +52,16 @@ database is on.
             - **Generate Client Certificate** and configure your client to use
                 the generated certificate.
             - Paste the certificate for your client.
-            For more information, see [Securing Redis Client Connections]({{< relref "/rs/administering/designing-production/security/client-connections.md" >}})
-    1. Choose a **Data Eviction Policy** or accept the default. For more information,
-        see [Data Eviction Policies]({{< relref "rc/concepts/data-eviction-policies.md" >}}).
-    1. Enable **Periodic Backups** and specify the [database backup]({{< relref "/rc/administration/configuration/backups.md" >}}) location.
-    1. Choose a **Module** to use with the database. If you
-        select 'RediSearch', enter the estimated number of documents
-        you want to index. If you select 'RedisGraph', calculate the
-        required resources with the [sizing calculator](https://redislabs.com/redis-enterprise/redis-graph/redisgraph-calculator/).
-    1. In **Alert Settings**, select the alerts that you want sent to you and
-        your team when the specified threshold is passed.
+    - **Data Eviction Policy** - Select a policy for evicting data when the memory limit is reached.
```

Comment:
s/memory limit/max memory limit/

Reply:
Isn't the memory limit the max? Do we need to say "max"?